src/test/java/com/flightstats/hub/webhook/WebhookRetryerTest.java

```diff
@@ -16,6 +18,7 @@
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 @ExtendWith(MockitoExtension.class)
+@Execution(ExecutionMode.SAME_THREAD)
```

Comment:
I think you'd mentioned why you had to add this mode, but I don't remember the failure/error you encountered. Was it a mockito issue?

Reply:
I think I found the problem, another unrelated but heavily mocked test had an unnecessary stub that caused mockito to throw (though it only threw the error sometimes, depending on random test order) removing that stub fixed these failing indeterminately in concurrent mode